src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -80,7 +77,11 @@ public void storeMessage(Message message) {
 
     return messages;
   }
-  
+  /**<p> Fetch a list of users based on all of the messages stored in Dataset</p>
```

Comment:
No need to use the <p> tags here 🙂 The documentation generator will take care of all of that formatting for you!